packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -17,17 +17,18 @@ import { grid } from '@bbc/psammead-styles/detection';
 
 import Link from '../Link';
 
-// Gets the number of grid rows, taking into account the
-// first-child in the grid being separate, on its own row.
-const getRowCount = (children, columns) =>
-  Math.ceil((React.Children.count(children) - 1) / columns) + 1;
+const getRowCount = (links, columns) =>
+  Math.ceil((links.length - 1) / columns) + 1;
 
 const StyledList = styled.ul`
-  border-bottom: 1px solid ${C_SHADOW};
+  border: 0.0625rem solid ${C_SHADOW};
```

Comment:
Since you're setting the `border-width` below, you don't need to set it here to then immediately overwrite it. 
```suggestion
  border: solid ${C_SHADOW};
```

Reply:
resolved